lib/marked.js

```diff
@@ -1066,7 +1066,7 @@ Renderer.prototype.codespan = function(text) {
 
 Renderer.prototype.inlinelatex = function(text) {
   //Return a helper object that is replaced inside of mattermost.
-  return `<span data-inline-latex="${escape(text)}"></span>`;
+  return '<span data-inline-latex="${escape(text)}"></span>';
```

Comment:
can you please explain this change from ` to '

Reply:
Why did it not complain before?

Reply:
I don't know, I think a lot of time passed since the last build.

Reply:
you can read it [here](https://pastebin.com/tTAThAEG)

Reply:
I'll pull this locally and see if I can sort out the issue since this change would definitely break inline latex support

Reply:
Maybe you should install uglifyjs globally. Anyway, I revered the changes as requested